spec/recipes_spec/books/organic-chemistry/expected_output.xhtml

```diff
@@ -2021,19 +2021,14 @@ To complete the structure of <span data-type="term" class="no-emphasis" id="auto
     Organic molecules are usually drawn using either <strong>condensed structures</strong> or skeletal structures. In condensed structures, carbon&#x2013;carbon and carbon&#x2013;hydrogen bonds aren&#x2019;t shown. In <strong>skeletal structures</strong>, only the bonds and not the atoms are shown. A carbon atom is assumed to be at the ends and at the junctions of lines (bonds), and the correct number of hydrogens is supplied mentally.
   </p>
           <div data-type="note" class="working-problems" id="auto_e8416a43-c719-4db7-a87a-1ee7ec8bb673_note-00002">
```

Comment:
@LostMojo The organic chemistry input content should be updated, because now it is misleading when the class is still there.